kalite/coachreports/static/js/coachreports/student_progress.js

```diff
@@ -101,10 +102,24 @@ var StudentProgressContainerView = Backbone.View.extend({
 
     initialize: function() {
         this.listenTo(this.collection, 'add', this.add_one);
+        this.count=0;
+        var self=this;
 
         this.render();
 
-        this.collection.fetch();
+        this.collection.fetch({
+                success: function() {
+                   
+                    self.count=self.collection.length;    //number of reports of quizes/videos
+                
+                     if(self.count==0)                    //if the student visits the my progress page before attempting any quizes/videos
+                          {
+				self.$el.html("<h>You have not started your learning journey yet.</h><br>Click on the LEARN button above to 						get started");
+				 
+                          }
+                    
+                }
+            });
```

Comment:
This isn't consistent with the style of the rest of the file. Try to avoid unnecessary newlines and expand your tabs to 4 spaces (but not more!).